etc/requirements.txt

```diff
@@ -1,6 +1,6 @@
 flask>=0.10.1
 flask-cors
-odo>=0.5.0
+# odo>=0.5.1
```

Comment:
I've made the pull request (see below). 

Once you change this, I'll revert this change and push the update. 

(For reasons I don't fully understand, pulling from my repo kept the version number below 0.5.0 so this kept failing :-/)